lnbits/decorators.py

```diff
@@ -252,8 +287,8 @@ async def check_admin(usr: UUID4) -> User:
     return user
 
 
-async def check_super_user(usr: UUID4) -> User:
-    user = await check_admin(usr)
+async def check_super_user(req: Request, usr: Optional[str] = None) -> User:
+    user = await check_admin(req, usr)
```

Comment:
- why not use UUID4 for usr?
```suggestion
async def check_super_user(req: Request, usr: Optional[UUID4] = None) -> User:
```